### DISCUSSION TITLE
":guilabel:" is displayed in Weblate Documentation.

### DISCUSSION THREAD

**Discussion by baffclan** [WeblateOrg/weblate#5406] [General] [answered]
Open Supported file formats , CSV files
https://docs.weblate.org/en/latest/formats.html#csv-files

![Supported file formats - CSV files](https://user-images.githubusercontent.com/18899730/107753937-1f0c7780-6d64-11eb-9fb1-780dcd2e0a00.png)

**Comment by nijel** [comment, marked as answer]
Thanks for reporting this, it should be fixed in 323403eaef0e5d9d0ab7214466bda3e8a999fb67.

**Reply by baffclan** [reply]
Thanks for fixing this!
